server/routes/induction/create/qualificationsListCreateController.ts

```diff
@@ -1,16 +1,12 @@
 import { NextFunction, Request, RequestHandler, Response } from 'express'
 import type { InductionDto } from 'inductionDto'
 import QualificationsListController from '../common/qualificationsListController'
-import { buildNewPageFlowHistory, getPreviousPage } from '../../pageFlowHistory'
+import { buildNewPageFlowHistory } from '../../pageFlowHistory'
 import getDynamicBackLinkAriaText from '../dynamicAriaTextResolver'
 
 export default class QualificationsListCreateController extends QualificationsListController {
   getBackLinkUrl(req: Request): string {
```

Comment:
@leejacobson-moj 
You can copy this `getBackLinkUrl` method and use it in your PR for Reasons Not To Get Work; as both pages (Reasons Not To Get Work and Qualifications List) are the next page from "Do you want to work on release" depending on how you answer it. IE. both pages need to go back to "Do you want to work on release" 👍